### PR TITLE
Mozilla bug 1861068 - Make nsHTML5AttributeName constructors not allocate

### PR DESCRIPTION
**Pull request by emilio**

### CODE REVIEW

**Review by hsivonen — CHANGES_REQUESTED** [review]
The fourth name (`l3`/`htmlLang`) should not show up on the C++ side.

File: src/nu/validator/htmlparser/impl/AttributeName.java
Comment (emilio) [reply src/nu/validator/htmlparser/impl/AttributeName.java]: Done.

File: translator-src/nu/validator/htmlparser/cpptranslate/CppVisitor.java
Comment (emilio) [reply translator-src/nu/validator/htmlparser/cpptranslate/CppVisitor.java]: Both, right? Both are unused in C++.

**Review by hsivonen — APPROVED** [review]
Thanks!

(I'll merge after the Gecko change lands.)

**Pull request merged**
